fares-data-build-tool/create-data
# feat: CFDS-44 - Line references

## Description

Updating line id from op:${service.lineName}#${service.serviceCode} to just lineId

## Testing instructions

Check XML to make sure **no** line ids are in this format op:${service.lineName}#${service.serviceCode}
